File: MediCareWebApi/Models/Feedback.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MediCare.Models
{
	public class Feedback
	{
		[Required]
		public DateTime CreatedAt { get; set; }

		[Required]
		[MaxLength(256)]
		public string Description { get; set; }

		[Range(1, 5)]
		public byte Rate { get; set; }

		[Key]
		public int AppointmentId { get; set; }

		[ForeignKey("AppointmentId")]
		public Appointment Appointment { get; set; }
	}
}

File: MediCareWebSpa/src/app/DTOs/models/feedback.ts
import { Appointment } from "./appointment";
import { Patient } from "./patient";

export class Feedback {
  id: number;
  createdAt: Date;
  description: string;
  rate: number;
  patient: Patient;
  appointmentId: number;
  appointment: Appointment;

  constructor(
    id: number,
    createdAt: Date,
    description: string,
    rate: number,
    patient: Patient,
    appointmentId: number,
    appointment: Appointment
  ) {
    this.id = id;
    this.createdAt = createdAt;
    this.description = description;
    this.rate = rate;
    this.patient = patient;
    this.appointmentId = appointmentId;
    this.appointment = appointment;
  }
}
